### ISSUE TITLE
last time \"t\" parameter must be specified in query string

### ISSUE THREAD

**Issue by sw3dish** [umdio/umdio#18]
Running into an issue with many bus-related methods where the resulting response contains a key as follows:

```
"Error":{"content":"last time \"t\" parameter must be specified in query string","shouldRetry":"false"}
```

**Comment by rrcobb** [comment]
That's an error we're getting passed by the NextBus API. It'll be addressed when we do an update on the way we deal with buses, but that might be a little while, unless someone steps up!

**Comment by sw3dish** [comment]
Maybe I'll look into it... Would give me an excuse to look into Ruby + Sinatra

**Comment by McIntireEvan** [comment]
Looking into this soon, not sure if the bus api still behaves the same way two and a half years later, so hopefully this resolved itself

**Comment by McIntireEvan** [comment]
See https://www.nextbus.com/xmlFeedDocs/NextBusXMLFeed.pdf for more info

**Comment by McIntireEvan** [comment]
Resolved by #68
